crates/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -46,13 +52,14 @@ pub fn insert_packages(
             debug!("editing existing [install] table");
             if let Item::Table(ref mut installs) = existing_installs.get_mut() {
                 for pkg in pkgs {
+                    debug!("checking for presence of package '{pkg}'");
                     if !installs.contains_key(&pkg) {
                         installs.insert(&pkg, Item::Value(Value::InlineTable(InlineTable::new())));
                         already_installed.insert(pkg.clone(), false);
-                        debug!("package {pkg} newly installed");
+                        debug!("package '{pkg}' newly installed");
                     } else {
                         already_installed.insert(pkg.clone(), true);
-                        debug!("package {pkg} already installed");
+                        debug!("package '{pkg}' already installed");
                     }
```

Comment:
suggestion: include in install PR

Reply:
It's already included here, I don't see a point in changing the other PR when this will be merged immediately afterwards.